scale-tests/configs/2018-08-22-testdrf112s.env

```diff
@@ -0,0 +1,129 @@
+# Depends on:
+# - TEST_NAME
+# - TEST_S3_BUCKET
+# - TEST_S3_FOLDER
+
+# Cluster configuration ########################################################
+
+CLUSTER_URL="https://testdrf112s.scaletesting.mesosphe.re"
```

Comment:
did this work on the test cluster? I think I remember reading that the test cluster only had 3 masters, 3 agents? Last time the cluster was so small it was not possible to verify the entire small-scale script could run to completion.

Reply:
It did not. The cluster had 10 agents, but they only had 4 cpus each, so I was only able to install ZK+Kafka+Cassandra, no space left for Spark dispatchers.